Android/Google.Android.DataTransport/source/Xamarin.Google.Android.DataTransport.TransportRuntime/Transforms/Metadata.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<metadata>
+
+    <attr
+        path="/api/package[@name='com.google.android.datatransport']"
+        name="managedName"
+        >
+        Xamarin.Google.Android.DataTransport
```

Comment:
Remove the Xamarin. prefix

Reply:
GOTO https://github.com/xamarin/XamarinComponents/pull/726/files/98e72486fa6118e860b870d9080b3acfe7ef5880#r339295307